packages/dds/tree/src/feature-libraries/sequence-field/moveEffectTable.ts

```diff
@@ -176,12 +177,12 @@ function getFirstMoveEffectLength(
 	effects: MoveEffectTable,
 ): number {
 	if (isMoveMark(markEffect)) {
-		return getMoveEffect(
-			effects,
+		return effects.get(
 			getCrossFieldTargetFromMove(markEffect),
 			markEffect.revision,
 			markEffect.id,
 			count,
+			true,
```

Comment:
This change seems unrelated to the PR. Was there a bug here?

Reply:
This was just needed to add the assert above. I can do this in a separate PR.

Reply:
In case that wasn't clear, I'm specifically asking about the addDependency argument `true` being added here.
I see now that addDependency was implicitly true before. So adding `true` explicitly doesn't change the behavior. I'm confused as to what adding that implicit param has to do with the assert though.

Reply:
Feel free to ignore this thread since you've reverted those changes.